bin/blazepack.js

```diff
@@ -196,6 +196,15 @@ if (args.version) {
       cloneSandbox({ id: sandboxId });
       break;
     }
+    case "help": {
+      const command = args._[1];
```

Comment:
since logHelp already handles the possibility of undefined should we remove this if case here?

Reply:
ohh yes, didn't saw that 😅